### DISCUSSION TITLE
Visualize single RBCs/PLTs out of all the cells

### DISCUSSION THREAD

**Discussion by mojtabaAmiraslanpour** [UvaCsl/HemoCell#101] [General discussion]
Hi,

I need to extract and see the motions of single RBCs or PLTs out of the many, in paraview. I am aware that I can extract a plot of the center point of these cells but currently, I am particularly interested in seeing the animations of the interactions between particular cells in a suspension.

Is this possible?

**Comment by gzavo** [comment]
Hi @mojtabaAmiraslanpour,

That is an interesting use-case, and should be possible, however it is not implemented out-of-the-box yet.
Some pointers:
- In the h5 files there is a unique identifier for cells “cellIds”
- there is an older script that shows how to access data based on cellId from python (scripts/hemocell_h5utils.py)

So you would need to do some postprocessing that isolates data based on specific cellIDs.

**Comment by ErenC1** [comment]
There is a easy way to do it. create RBC_i (as many as you want to track), set their outputs as usual. But if you are working with large hematocrit, initializing it might be tricky.

**Reply by mojtabaAmiraslanpour** [reply]
Well, I just used the Threshold filter in Paraview for the 'Cell Id' field and everything is good now. Thanks for the suggestion though.
